partiql-lang/src/test/kotlin/org/partiql/lang/eval/EvaluatingCompilerIntTest.kt

```diff
@@ -145,6 +145,20 @@ class EvaluatingCompilerIntTest : EvaluatorTestBase() {
         expectedPermissiveModeResult = "MISSING"
     )
 
+    @Test
+    @Parameters
+    fun bitwiseAnd(pair: Pair<String, String>) = assertPair(pair)
+
+    fun parametersForBitwiseAnd(): List<Pair<String, String>> {
+        val parameters = mutableListOf<Pair<String, String>>()
+
+        parameters.add("1 & 2" to "0")
+        parameters.add("3 & 5" to "1")
+        parameters.add("5 & 7" to "5")
+
```

Comment:
Could have another test w/ chained bitwise and e.g. `31 & 15 & 7 & 3 & 1` -> `1`